### DISCUSSION TITLE
Can be compiled in arm64

### DISCUSSION THREAD

**Discussion by ixxmi** [duckdb/duckdb#7202] [Ideas]
Can be compiled in arm64, please help me,thanks

**Comment by hannes** [comment]
Some more details please. In general, DuckDB can be compiled for arm64 on various platforms.

**Reply by ixxmi** [reply]
I am using golang for cross-compilation, the system architecture is linux arm64, I don't know how to compile, I hope I can continue to use duckdb, because I am not familiar with c++, so I don't know the specific operation, is there a compiled version?

**Comment by hannes** [comment]
Yes, there are arm builds for linux, e.g. here https://github.com/duckdb/duckdb/releases/download/v0.7.1/libduckdb-linux-aarch64.zip and https://github.com/duckdb/duckdb/releases/download/v0.7.1/duckdb_cli-linux-aarch64.zip

**Reply by ixxmi** [reply]
thanks
